backend/tests/test_sessions_routes.py

```diff
@@ -274,6 +277,34 @@ def test_no_readonly_session_as_user(client, db, user, kubernetes):
     assert not kubernetes.sessions
 
 
+def test_create_readonly_session_as_user(client, db, user, kubernetes):
+    _tool, version = next(
```

Comment:
```suggestion
    _, version = next(
```

`_tool` is not used.

Reply:
I left it as `_tool`, so you know what it actually was. I think that makes it more clear to the user. `_` doesn't give you any information.